Sources/Serveable.swift

```diff
@@ -0,0 +1,10 @@
+
+/// Serves your order to a local or a remote bar
+
+public protocol Serveable {
+
+    associatedtype S
+
+    func serve(order: Order, delivery: (Delivery<S>)->())
```

Comment:
syntax: Yes indeed. Still the deal. I was just preparing service to do just that but forgot to use service in Bar. Will do this now.